DemocracyClub/WhoCanIVoteFor
# Feature/photo id wording

This work adds voter ID messaging to both the front page and postcode lookup pages. [Relevant Trello card](https://trello.com/c/UvMnJkuS/3207-voter-id-messaging).
This messaging is the same as that which was added to [WDIV](https://github.com/DemocracyClub/UK-Polling-Stations/pull/5275).

To test:
- Run Who locally. 
    - I'd recommend updating your local data so that you've got the upcoming May elections. 
    - Make sure you do the migration too 😉 
- On the homepage you should see an info box on the front page.
- Pick any postcode you know to have upcoming elections. You should see a card "You will need to take photo ID..." as in the below image.

**Postcode lookup:** 

<img width="962" alt="Screenshot 2023-03-23 at 11 01 52" src="https://user-images.githubusercontent.com/9531063/227183843-faaf2339-5520-4f85-b1fb-b620bd7284e4.png">


**Front page:** 

<img width="966" alt="Screenshot 2023-03-23 at 10 53 35" src="https://user-images.githubusercontent.com/9531063/227181997-74dbc42e-4567-4a91-99ab-a9085d066965.png">


```[tasklist]
### PR Checklist

- [x] No unintentional console.logs left behind after debugging

```
